### PR TITLE
Add RubyMotion

### PR DESCRIPTION
**Pull request by fengb**

### CODE REVIEW

File: Casks/rubymotion.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask :v1 => 'rubymotion' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://www.rubymotion.com/files/RubyMotion%20Installer.zip'
+  name 'RubyMotion'
+  homepage 'http://www.rubymotion.com/'
+  license :commercial
+
+  installer :script => 'RubyMotion Installer.app/Contents/MacOS/RubyMotion Installer'
```
Comment (vitorgalvao) [inline Casks/rubymotion.rb]: Won’t this call a GUI?

**Comment by fengb** [comment]
Is that a problem? The installer needs to run and I saw a few other casks using this invocation.

**Comment by vitorgalvao** [comment]
Yes. If other apps are using this technique, they’re either cases that do not call a GUI, or are wrong, in which case we should check them.

This should use [something else](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#installer-manual). It’s also missing `uninstall`. There are [scripts](https://github.com/caskroom/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#uninstall-stanza-details) geared for that.

**Comment by fengb** [comment]
Alright, I've updated the cask to be more standardized.

I'm a bit curious about the "manual" convention. When setting up a new box, I tend to chain commands together and  manual output gets ignored.

**Comment by vitorgalvao** [comment]
`:manual` doesn’t perform any actions, it only warns you what you have to run manually (hence the name).

**Pull request merged**